apis/logging/v1/clusterlogging_types.go

```diff
@@ -506,6 +528,7 @@ type LogStoreType string
 
 const (
 	LogStoreTypeElasticsearch LogStoreType = "elasticsearch"
+	LogStoreTypeLokiStack     LogStoreType = "lokiStack"
```

Comment:
To avoid a probably common typo with the capital `S` I suggest to use `lokistack` as the constant value here.